packages/images-plugin/lib/plugin.js

```diff
@@ -83,6 +82,36 @@ export default class ImageElementPlugin extends BasePlugin {
     // otherwise just do the normal thing
     return await next();
   }
+
+  async performActions(next, driver, ...args) {
+    // Replace with coordinates when ActionSequence includes image elements.
+    const [actionSequences] = /** @type {[import('@appium/types').ActionSequence[]]} */ (args);
+    for (const actionSequence of actionSequences) {
+      for (const action of actionSequence.actions) {
+        // The actions that can have an Element as the origin are "pointerMove" and "scroll".
+        if (
+          (action.type === 'pointerMove' || action.type === 'scroll') &&
+          !_.isNil(action.origin) &&
```

Comment:
perhaps it would make more sense to only verify `_.isPlainObject(action.origin)` instead of checking for nils and strings